### PR TITLE
DO NOT MERGE Add Elixir tutorial

### PR DESCRIPTION
**Pull request by jeffweiss**

### CODE REVIEW

File: site/tutorials/tutorial-two-elixir.md [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+When RabbitMQ quits or crashes it will forget the queues and messages
+unless you tell it not to. Two things are required to make sure that
+messages aren't lost: we need to mark both the queue and messages as
+durable.
+
+First, we need to make sure that RabbitMQ will never lose our
+queue. In order to do so, we need to declare it as _durable_:
+
+    :::elixir
+    AMQP.Queue.declare(channel, "hello", durable: true)
```
Comment (michaelklishin) [inline site/tutorials/tutorial-two-elixir.md]: "hello" should be "task_queue"

File: site/tutorials/tutorial-two-elixir.md [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+
+This `AMQP.Queue.declare` change needs to be applied to both the producer
+and consumer code.
+
+At that point we're sure that the `task_queue` queue won't be lost
+even if RabbitMQ restarts. Now we need to mark our messages as persistent
+- by supplying a `persistent: true` property.
+
+    :::elixir
+    AMQP.Basic.publish(channel, "", "hello", message, persistent: true)
```
Comment (michaelklishin) [inline site/tutorials/tutorial-two-elixir.md]: "hello" should be "task_queue"

**Comment by michaelklishin** [comment]
Please submit this against the `live` branch. `master` will only be deployed with RabbitMQ 3.7.0, more than 6 months from now.

**Comment by michaelklishin** [comment]
@jeffweiss thank you. This PR needs some work, please see the comments above.

**Comment by jeffweiss** [comment]
Noted. I'll make the changes are resubmit against the `live` branch.

**Comment by michaelklishin** [comment]
@bdshroyer when you have a chance (and get to other Elixir-related things we've discussed), feel free to finish this and merge. Website changes that we'd like to deploy quickly should be based on the `live` branch. `stable` is for `3.6.x` release (at the moment) and `master` is for the next feature release (`3.7.0` at the moment).

**Comment by michaelklishin** [comment]
I've corrected a few things in tutorial two. We now have enough tutorials that I had to rework our getting started guide ;) Thank you, @jeffweiss, it's [already live](http://www.rabbitmq.com/getstarted.html).

**Comment by michaelklishin** [comment]
Oh, and to push it live, I cherry-picked relevant commits from master.

**Comment by jeffweiss** [comment]
Sorry I've been non-responsive. I've really been underwater and ill lately. Thanks for making it happen.

**Comment by michaelklishin** [comment]
@jeffweiss no worries. Get well soon!

**Comment by jeffweiss** [comment]
💜

**Pull request merged**